server/app/lib/message.rb

```diff
@@ -0,0 +1,33 @@
+class Message
+  def self.not_found(record = 'record')
+    "Sorry, #{record} not found."
```

Comment:
just a note: usually it is a good idea to be consistent with code style and I am used to look for consistency. Because of this line I just learned that there is a difference between using double and single quotes in ruby. I found this interesting article comparing both: https://anti-pattern.com/always-use-double-quoted-strings-in-ruby 
Note that I am not asking you to make changes here - go for whatever you like. If you think it is interesting and valuable, feel free to put code conventions on the table for discussion by the backend team at one of your next meetings so that a style guide is chosen (I bet there are tools also to ensure some style rules are followed)